site_config_client/__init__.py

```diff
@@ -96,7 +99,9 @@ def get_config(self, site_uuid: Union[str, uuid.UUID],
             "name": name,
             "status": status
         }
-        response = requests.get(self.build_url(endpoint), data=data)
+        auth_headers = {'Authorization': 'Token {}'.format(self.api_token)}
+        response = requests.get(self.build_url(endpoint),
+                                headers=auth_headers, params=data)
```

Comment:
@OmarIthawi changed this from `data=data` ➡️  `params=data` to get it to work in prod